ci/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Python 3.8 is supported until 14 October 2024.
-# Alpine Linux 3.17 is supported until 22 November 2024.
-FROM python:3.8-alpine3.17
+# Alpine Linux 3.16 is supported until 23 May 2024.
```

Comment:
maybe we want to add a commented line about not updating to 3.17 until the urllib 2.0, botocore, requests thing is sorted?

Reply:
I added more comments.